cmake/templates/BoostConfig.cmake.in

```diff
@@ -103,10 +103,22 @@ if(TARGET Boost::iostreams)
   include("${CMAKE_CURRENT_LIST_DIR}/BoostBZip2Dependency.cmake" OPTIONAL)
 endif()
 
+include(CheckLibraryExists) # check_library_exists
+check_library_exists(rt clock_gettime "" _boost_check_rt_library)
```

Comment:
We should run checks only if we really need it. I.e. under condition `if(TARGET Boost::thread OR TARGET Boost::chrono)`.